### PR TITLE
Update Grafana To match new HA status metrics change

### PR DESCRIPTION
**Pull request by tmiddlet2666**

### CODE REVIEW

File: operator/src/main/helm/coherence-operator/dashboards/services-summary-dashboard.json [hunk 1]
```diff
@@ -196,12 +200,20 @@
...
-          "expr": "avg(coherence_partition_assignment_service_node_count{cluster=~\"$ClusterName\"}) by (coherence_service, tag_HA_status) ",
+          "expr": "avg(coherence_partition_assignment_service_node_count{cluster=~\"$ClusterName\"}) by (coherence_service) ",
           "format": "table",
           "instant": true,
           "intervalFactor": 1,
           "legendFormat": "",
           "refId": "A"
+        },
+        {
+          "expr": "avg(coherence_partition_assignment_HA_status{cluster=~\"$ClusterName\"}) by (coherence_service)",
```
Comment (jfialli) [inline operator/src/main/helm/coherence-operator/dashboards/services-summary-dashboard.json]: avg is misleading for HA_status.   It is a single enum for entire cluster.  Luckily only one node has value. 
Unsure what happens when change in coordinator if there is period of 2 nodes looking like they have the value. we really want one and only one HA_status per cluster, coherence_service.

File: operator/src/main/helm/coherence-operator/dashboards/coherence-dashboard-main.json [hunk 2]
```diff
@@ -410,7 +406,7 @@
       "tableColumn": "",
       "targets": [
         {
-          "expr": "count(avg(coherence_service_partitions_endangered{cluster=~\"$ClusterName\",tag_type=~\"DistributedCache|FederatedCache\"}) by(name) > 0)",
+          "expr": "count(coherence_partition_assignment_HA_status{cluster=~\"$ClusterName\"} == 1)",
```
Comment (jfialli) [inline operator/src/main/helm/coherence-operator/dashboards/coherence-dashboard-main.json]: does the above need a "by (coherence_serivce)" added to it like the HA_status reference later on in this file.

Also, not sure this should be a count, there should be only one HA_status per cluster,coherence_service metrics labels.

File: operator/src/main/helm/coherence-operator/dashboards/alerts-dashboard.json [hunk 3]
```diff
@@ -665,10 +667,11 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "count(coherence_cluster_size)",
+          "expr": "count(coherence_partition_assignment_HA_status == 1)",
```
Comment (jfialli) [inline operator/src/main/helm/coherence-operator/dashboards/alerts-dashboard.json]: we are returning 0 for orphaned and 1 for engandered.  getHAStatus() folded orphaned and endangered into being endangered. harvey wanted me to just return the actual backupstrength number and 0 represented that we already lost data and 1 meant we were at risk.  You could fold into one category by checking for less than or equal to zero. I return -1 if there is no value for mbean attribute HAStatusCode on a node and I believe -1 is filtered out.

File: operator/src/main/helm/coherence-operator/dashboards/services-summary-dashboard.json (hunk 1 above)
Comment (tmiddlet2666) [reply operator/src/main/helm/coherence-operator/dashboards/services-summary-dashboard.json]: Hi @jfialli , the reason for having avg is to allow me to group by coherence_service and therefore making integrating this metric easier into the existing panel. Having always being only one value for cluster and service, the avg is not really giving us an average.
Overall doing it this way makes it easier to fit the metric into the panel.
Hope this makes sense.

File: operator/src/main/helm/coherence-operator/dashboards/alerts-dashboard.json (hunk 3 above)
Comment (tmiddlet2666) [reply operator/src/main/helm/coherence-operator/dashboards/alerts-dashboard.json]: I was not aware 0 was for orphaned, so i can make <= 1, which will mean any endangered or orphaned.

File: operator/src/main/helm/coherence-operator/dashboards/coherence-dashboard-main.json (hunk 2 above)
Comment (tmiddlet2666) [reply operator/src/main/helm/coherence-operator/dashboards/coherence-dashboard-main.json]: The above expression is to show the total count of services that are in endangered status. I will make this one <= 1 as well. The by (coherence_service), won't be needed as this is a total.

File: operator/src/main/helm/coherence-operator/dashboards/services-summary-dashboard.json (hunk 1 above)
Comment (jfialli) [reply operator/src/main/helm/coherence-operator/dashboards/services-summary-dashboard.json]: Thanks for explaining it.

**Pull request merged**